### PR TITLE
Wmg/cbo rehire api

### PR DESCRIPTION
**Pull request by allthesignals**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,8 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+.env
+.DS_Store
```
Comment (mzagaja) [inline .gitignore]: These system files can/should be ignored in the global .gitignore

File: spec/factories/rehire_sites.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :rehire_site do
+
```
Comment (mzagaja) [inline spec/factories/rehire_sites.rb]: See https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories for how to define a Factory for a rehire site.

**Comment by allthesignals** [comment]
@mzagaja in order to QA whether the endpoint works, you need to do that in postman and set the `content type` and `accept` headers to `application/vnd.api+json` as per the JSON API standard: http://jsonapi.org/#mime-types. 

Will address the other changes. 

Thanks!

File: config/routes.rb [hunk 3]
```diff
@@ -2,5 +2,11 @@
   get 'travel_time/get'
   resources :position_imports, only: [:create]
 
+  scope 'api' do
+    resources :rehire_sites do
```
Comment (mzagaja) [inline config/routes.rb]: This should be restricted to the available actions for rehire_sites. See [routing concerns](http://guides.rubyonrails.org/routing.html#routing-concerns) for the syntax of using "only" with resources.

Comment (allthesignals) [inline config/routes.rb]: Sure - how come? Security? Only the define methods on the controller will work.

Comment (mzagaja) [inline config/routes.rb]: It's a message to others that use the program that you aren't using those routes. My typical workflow is something like get the pull request, run `rake routes` and see the list of available routes. Then test each endpoint. At that point if you use resources without only then it shows me all the endpoints as being available and then I don't realize there's nothing there to test until I peek into the controller.

Comment (allthesignals) [inline config/routes.rb]: Aaah - thank you.

**Pull request merged**